server/views/pages/reviewPlan/review/checkYourAnswers/index.test.ts

```diff
@@ -41,7 +41,7 @@ describe('ReviewPlanCheckYourAnswersPage', () => {
     expect($('[data-qa="review-completed-by-change-link"]').attr('href')).toBe(
       `/plan/${prisonerSummary.prisonNumber}/review`,
     )
-    expect($('[data-qa="review-completed-by-MYSELF"]').text().trim()).toBe('I completed the review myself')
+    expect($('[data-qa="review-completed-by-MYSELF"]').length).toEqual(1)
```

Comment:
Make the test less fragile - what we care about is that the element is rendered, not what the specific content is